dll/win32/mpr/CMakeLists.txt

```diff
@@ -23,6 +23,6 @@ add_library(mpr SHARED
 
 set_module_type(mpr win32dll)
 target_link_libraries(mpr wine)
-add_importlibs(mpr advapi32 user32 msvcrt kernel32 ntdll)
+add_importlibs(mpr advapi32 netplwiz user32 msvcrt kernel32 ntdll)
```

Comment:
I don't think `mpr` is supposed to have a `netplwiz` import?

Reply:
Should be taken care of in my fix for HBelusca LoadLibrary request.